packages/registry/profiles/microsoft/microsoft-045e-065d.json

```diff
@@ -0,0 +1,32 @@
+{
+    "profileId": "microsoft-045e-065d",
+    "fallbackProfileIds": [ "generic-trigger-grip-touchpad-thumbstick"],
```

Comment:
Shouldn't this have the "b" variant in the fallback list? That's like our canonical example of when a non-generic profile is useful.

Reply:
d'oh.  copy paste fail.